### DISCUSSION TITLE
Synchronize keyboard input to multiple splits

### DISCUSSION THREAD

**Discussion by YLivay** [ghostty-org/ghostty#4046] [Feature Requests, Ideas] [answered]
In [Tilix](https://github.com/gnunn1/tilix) there is a feature to synchronize keyboard input so whatever you type in one panel is replicated to the other panels within the same OS window.

In Kitty there is a similar feature using the [broadcast kitten](https://sw.kovidgoyal.net/kitty/kittens/broadcast/)

Is there an equivalent feature in Ghostty? I do a lot of devops which requires sending the same or very similar commands to multiple destinations this feature is a huge time saver and i'd love to see it in Ghostty!

Ideally it would be implemented like in Tilix where its a mode you can toggle with a keyboard shortcut and you type directly in any split.

**Comment by JakeWharton** [comment, marked as answer]
Dupe of https://github.com/ghostty-org/ghostty/discussions/3227

**Reply by YLivay** [reply]
Thanks! I did search before opening this discussion but the other topic didn't pop up.
